src/engine/bank1.asm

```diff
@@ -580,17 +678,17 @@ Func_4823: ; 4823 (1:4823)
 	push hl
 	push bc
 	ld e, b
-	ld hl, $cc34
+	ld hl, wCardBuffer1 + $10
```

Comment:
Do you think you can work on wram constants for card data fields? For example this would be `wCardBuffer1 + (wCardBuffer1Move1Name - wCardBuffer1Move1)`, and the `ld hl, $0006` a few lines below would be `wCardBuffer1Move1Category - (wCardBuffer1Move1Name + 1)`.